mytoyota/models/endpoints/trips.py

```diff
@@ -8,6 +10,19 @@
 from mytoyota.models.endpoints.common import StatusModel
 
 
+def add_with_none(this, that):
```

Comment:
Could you please explain what exactly this is needed for.
Please also add a few unit tests to maintain the coverage. I think these could also contribute to the explanation.

Reply:
When creating the `__add__` functions I found myself writing this set of code over and over to deal with the situation where either this and/or that are None. If either side is None(but not both) you get an exception. In these situations we just want the result to be the side that is not None.

Reply:
As far as I know, there is no built-in python functionality for this. I would have solved it slightly differently, but this is more or less similar to your approach:

```python
def add_with_none(a: Any, b: Any) -> Any:
    return a + b if a is not None and b is not None else a or b
```
Since the function can be used for anything, I still think that it is better placed somewhere in the `utils` than in the trips endpoints models.

I generally think that some more specific unit tests can already be helpful in the work before we move on to writing larger integration tests. :blush: